tests/ZendService/Apple/Apns/MessageTest.php

```diff
@@ -115,6 +115,19 @@ public function testSetSoundThrowsExceptionOnNonString()
         $this->message->setSound(12345);
     }
 
+    public function testSetContentAvailableThrowsExceptionOnNonInteger()
+    {
+        $this->setExpectedException('InvalidArgumentException');
+        $this->message->setContentAvailable("string");
+    }
+
+    public function testGetContentAvailableReturnsCorrectInteger()
+    {
+        $value = 1;
```

Comment:
No, just about the apple push notifications allowed values for `content-available`